packages/components/src/TabBar/TabBar.component.js

```diff
@@ -6,12 +6,26 @@ import Tab from 'react-bootstrap/lib/Tab';
 import Nav from 'react-bootstrap/lib/Nav';
 import NavItem from 'react-bootstrap/lib/NavItem';
 import keycode from 'keycode';
+import debounce from 'lodash/debounce';
+import classnames from 'classnames';
+import theme from './TabBar.scss';
 
 class TabBar extends React.Component {
 	constructor(props) {
 		super(props);
 		this.handleSelect = this.handleSelect.bind(this);
 		this.handleKeyDown = this.handleKeyDown.bind(this);
+		this.shouldShowDropdown = this.shouldShowDropdown.bind(this);
+		this.showNavBarAndTest = this.showNavBarAndTest.bind(this);
+		this.state = {
+			showDropdown: false,
+		};
+		this.navBarContainerRef = React.createRef();
```

Comment:
For consistency, I'd rather have "tabBar" used everywhere it references the TabBar component
```suggestion
		this.tabBarContainerRef = React.createRef();
```